tests/bgp/test_bgp_chasssis.py

```diff
@@ -0,0 +1,424 @@
+                                      if neighbor.endswith('T1')])
+            for nbr in t1_neighbors:
+                nbr_offset = tbinfo['topo']['properties']['topology']['VMs'][nbr]['vm_offset']
+                t1_exabgp_ports[IPV4][nbr] = EXABGP_BASE_PORT + nbr_offset
+                t1_exabgp_ports[IPV6][nbr] = EXABGP_BASE_PORT_V6 + nbr_offset
+
+            for dut in duthosts:
+                if dut.is_supervisor_node():
+                    continue
+                cfg_facts = all_cfg_facts[dut.hostname][0]['ansible_facts']
+                neighbors = cfg_facts['BGP_NEIGHBOR']
+                bgp_neighbors[IPV4][dut] = []
+                bgp_neighbors[IPV6][dut] = []
+                for neigh in neighbors.keys():
+                    ipversion = ipaddress.ip_address(neigh).version
+                    bgp_neighbors[ipversion][dut].append(neigh)
+                pytest_assert(len(bgp_neighbors[IPV4][dut]) >= 2,
+                              "Need atleast two v4 neighbors for the test")
+                pytest_assert(len(bgp_neighbors[IPV6][dut]) >= 2,
+                              "Need atleast two v6 neighbors for the test")
+
+        def verify_nexthops(nexthops, expected_nexhops):
+            nexthops.sort()
+            expected_nexhops.sort()
+            return nexthops == expected_nexhops
+
+        def verify_ip_route(duthost, prefix, expected_nexhops=None):
+            ip_route = duthost.get_ip_route(prefix)
+            if expected_nexhops is None:
+                if ip_route.keys():
+                    return True
+                return False
+
+            nexthops = []
+            for _, routes in ip_route.items():
+                for route in routes:
+                    for nexthop in route['nexthops']:
+                        nexthops.append(nexthop['ip'])
+
+            return verify_nexthops(nexthops, expected_nexhops)
+
+        def verify_bgp_route(duthost, prefix, expected_nexhops=None,
+                             nexthops_with_inferior_paths=None):
+            route = duthost.get_bgp_route_info(prefix)
+            if expected_nexhops is None:
+                if route.keys():
+                    return True
+                return False
+
+            nexthops = []
+            inferior_nexthops = []
+            if 'paths' not in route:
+                return False
+            for path in route['paths']:
+                if not path['valid'] or 'bestpath' not in path or \
+                    'nexthops' not in path:
+                    if 'nexthops' in path and path['valid']:
+                        for nexthop in path['nexthops']:
+                            inferior_nexthops.append(nexthop['ip'])
+                    continue
+                for nexthop in path['nexthops']:
+                    nexthops.append(nexthop['ip'])
+
+            if nexthops_with_inferior_paths:
+                return (verify_nexthops(nexthops, expected_nexhops) and
```

Comment:
just update "nexthops" and "exptected_nexthops" and let the last statement take care of verifying.